aopy/visualization/base.py

```diff
@@ -2430,4 +2430,72 @@ def overlay_sulci_on_spatial_map(subject, chamber, drive_type, theta=0, center=(
     filename = f'{subject.lower()}_{chamber.lower()}_{drive_type.lower()}_sulci.png'
     params_file = as_file(config_dir.joinpath(filename))
     with params_file as f:
-        overlay_image_on_spatial_map(f, drive_type, theta=theta, center=center, alpha=alpha, **kwargs)
\ No newline at end of file
+        overlay_image_on_spatial_map(f, drive_type, theta=theta, center=center, alpha=alpha, **kwargs)
+
+def plot_plane(ax, plane, scale=1.0, color='grey', alpha=0.15, resolution=100):
+    """
+    Plots a 3D plane on a given Matplotlib axis.
+
+    This function generates a square plane centered at the origin, applies a transformation 
+    using a given mapping (either a rotation matrix or a plane equation), and renders it in 3D space.
```

Comment:
I think some of description is not relevant to this function
```suggestion
    Plots a 3D plane centered at the origin.
```